### ISSUE TITLE
[FEATURE] Add UniData Language Syntax

### ISSUE THREAD

**Issue by itsxallwater** [mvextensions/mvbasic#13]
**Is your feature request related to a problem? Please describe.**
Currently, the `client\Syntaxes\MvLanguage.json` file contains MVON# Basic language definition and the `client\Syntaxes\MVON.tmLanguage.json` file contains the MVON# Basic syntax highlighting definition. We are looking to include UniData syntax out of the box to have available as an option within the extension.

**Describe the solution you'd like**
An acceptable pull request for this issue would include a UniData versions of the MvLanguage.json and MVON.tmLanguage.json files that can be included in the base package. This pull request need not concern itself with any of the plumbing for the extension to be able to dynamically load that syntax into place, we're simply looking for the syntax definition here.

**Comment by ianmcgowan** [comment]
I'd like to take a stab at this - I know Unidata, but not javascript or the internals of VSCode.  Many (but not all) of the Unidata keywords will be similar to Universe and HELP.FILE can be reviewed to jump start the effort.

**Comment by ianmcgowan** [comment]
After looking around at this, it seems like some kind soul has done most of the hard work already in this [snippets-unidata.json](https://github.com/mvextensions/mvbasic/blob/master/Syntaxes/snippets-unidata.json) file for Unidata.  Would it be ok to transform that data into UniDataLanguage.json?  The main thing missing is having different icons for different types of commands.

**Comment by itsxallwater** [comment]
I think so, @ianmcgowan! If memory serves these came about via @andrewcole50.

**Comment by andrewcole50** [comment]
> I think so, @ianmcgowan! If memory serves these came about via @andrewcole50.

Yep that was me! Important note: I'm not a Unidata guy and everything was straight from the documentation. With that, I'm sure things could be worded better and there are probably more realistic examples that could be used.

**Comment by andrewcole50** [comment]
@ianmcgowan I wrote a converter script that converted the snippets files to the {FLAVOR}Language.json files for d3 and I thought I'd run it for UniData as well. The script made estimates on which icons to use and I did NOT manually look at much of the results. I think it would be great if you could look at the referenced PR and verify what was done.

AFAIK, the icon numbers correlate to:
3 = Function
14 = Statement
21 = CONSTANT
